roles/node_prep/tasks/10_validation.yml

```diff
@@ -348,7 +348,9 @@
 - name: Fail if dual-stack is requested and build does not support it
   fail:
     msg: This release {{ release_version }} does not support dual-stack deployments
-  when: release_version.split('.')[0]|int == 4 and release_version.split('.')[1]|int < 6 and dualstack_baremetal
+  when:
+    - release_version is ansible.builtin.version('4.6', '<')
+    - dualstack_baremetal
```

Comment:
now that you are making easier to read these conditions, can we be more explicit what are we checking with dualstack_baremetal? Maybe: `dualstack_baremetal | bool`, wdyt?

Reply:
100% agree, done